app/views/dashboards/_publisher.html.erb

```diff
@@ -39,7 +45,7 @@
           <span class="fas fa-eye btn-icon__inner"></span>
         </span>
         <div class="media-body">
-          <span class="d-block font-size-3">TBD</span>
+          <span class="d-block font-size-3"><%= number_with_delimiter current_user.impressions_as_publisher.between(@start_date, @end_date).count %></span>
```

Comment:
Would prefer to see such queries defined in the controller or possibly a helper.